### ISSUE TITLE
Looks like some apps aren't themed anymore

### ISSUE THREAD

**Issue by OlaoluwaM** [catppuccin/gtk#258]
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### Describe the bug.

Some gnome apps are no longer themed with the latest release. These apps were themed in the past release, though

### Latest Release:

![image](https://github.com/catppuccin/gtk/assets/37044906/2bc81471-fd1b-4ef0-a2bc-68179b958c84)

![image](https://github.com/catppuccin/gtk/assets/37044906/2a21c836-10a0-4416-84a3-a37bac4f25f7)

### Previous Release:

![image](https://github.com/catppuccin/gtk/assets/37044906/f106d04d-59d1-452e-8c0c-ae0f2b35c849)

![image](https://github.com/catppuccin/gtk/assets/37044906/fcdd4834-a956-4948-9ed0-8ead10af9b80)

### How did you install the theme?

From release (manual installation)

### Are you running the latest release, and if not does the issue persist there?

- [X] I am running on the latest release & the issue persists there

### If using GTK4, have you symlinked the "gtk-4.0" folder?

- [X] I have symlinked the `gtk-4.0` folder.

### What GNOME version are you seeing the bug on?

GNOME Shell 46.2

### Any additional comments?

Everything else looks good. Great work!

**Comment by isabelroses** [comment]
Can you check if this issue is any similar to https://github.com/catppuccin/gtk/issues/255

**Comment by OlaoluwaM** [comment]
No, it doesn't seem related. The symlinks look correct:

```sh
total 28K
drwx------. 1 olaolu olaolu  100 May 28 07:01 .
drwxr-xr-x. 1 olaolu olaolu 1012 May 28 06:37 ..
lrwxrwxrwx. 1 olaolu olaolu   87 May 28 07:01 assets -> /home/olaolu/.local/share/themes/catppuccin-mocha-lavender-standard-dark/gtk-4.0/assets
lrwxrwxrwx. 1 olaolu olaolu   88 May 28 07:01 gtk.css -> /home/olaolu/.local/share/themes/catppuccin-mocha-lavender-standard-dark/gtk-4.0/gtk.css
lrwxrwxrwx. 1 olaolu olaolu   93 May 28 07:01 gtk-dark.css -> /home/olaolu/.local/share/themes/catppuccin-mocha-lavender-standard-dark/gtk-4.0/gtk-dark.css
-rw-r--r--. 1 olaolu olaolu   47 May 28 06:56 settings.ini
lrwxrwxrwx. 1 olaolu olaolu   94 May 28 07:01 thumbnail.png -> /home/olaolu/.local/share/themes/catppuccin-mocha-lavender-standard-dark/gtk-4.0/thumbnail.png
```

**Comment by OlaoluwaM** [comment]
Hmm, Is there a way to check? The affected applications seem to be those installed by default by Fedora

**Comment by OlaoluwaM** [comment]
Using `ldd` these apps are gtk-3 apps

**Comment by nullishamy** [comment]
Testing seems to indicate evince is a GTK3 application too, for context. It is affected by gnome-tweaks' Legacy Applications option.

**Comment by OlaoluwaM** [comment]
Hmm, it does work, but only if you execute the apps directly from the terminal. Try launching them as regular GUI apps and you'll see what I mean.

```sh
ls -lah /home/olaolu/.local/share/themes/catppuccin-mocha-lavender-standard-dark

total 4.0K
drwxr-xr-x. 1 olaolu olaolu  128 May 27 19:25 .
drwxr-xr-x. 1 olaolu olaolu  512 May 28 06:48 ..
drwxr-xr-x. 1 olaolu olaolu   62 May 27 19:25 cinnamon
drwxr-xr-x. 1 olaolu olaolu   64 May 27 19:25 gnome-shell
drwxr-xr-x. 1 olaolu olaolu   76 May 27 19:25 gtk-3.0
drwxr-xr-x. 1 olaolu olaolu   76 May 27 19:25 gtk-4.0
-rw-r--r--. 1 olaolu olaolu  378 May 27 18:03 index.theme
drwxr-xr-x. 1 olaolu olaolu   78 May 27 19:25 metacity-1
drwxr-xr-x. 1 olaolu olaolu   20 May 27 19:25 plank
drwxr-xr-x. 1 olaolu olaolu 2.4K May 27 19:25 xfwm4
```

I tried copying the contents of /home/olaolu/.local/share/themes/catppuccin-mocha-lavender-standard-dark/gtk-3.0` to `~/.config/gtk-3.0` and the result is this 🥲:

![image](https://github.com/catppuccin/gtk/assets/37044906/a6c0e36f-b04c-4687-b68b-c55217c220e4)

**Comment by OlaoluwaM** [comment]
> Testing seems to indicate evince is a GTK3 application too, for context. It is affected by gnome-tweaks' Legacy Applications option.

As a workaround, I'll use the previous release for the "Legacy Applications" option in gnome-tweaks for now

**Comment by nullishamy** [comment]
![image](https://github.com/catppuccin/gtk/assets/99221043/ddcde995-8d20-4616-b1fb-4fbeaa92ec48)
Works fine when launching from the app launcher to for me. I suspect your GNOME/GTK environment vars are not being set correctly

**Comment by OlaoluwaM** [comment]
> ![image](https://github.com/catppuccin/gtk/assets/99221043/ddcde995-8d20-4616-b1fb-4fbeaa92ec48)
> Works fine when launching from the app launcher to for me. I suspect your GNOME/GTK environment vars are not being set correctly

which envs would you suspect?

**Comment by OlaoluwaM** [comment]
I have the `GTK_THEME` env set btw

**Comment by OlaoluwaM** [comment]
[![Laughing_Facepalm](https://cdn3.emoji.gg/emojis/5084-laughing-facepalm.png)](https://emoji.gg/emoji/5084-laughing-facepalm)

**Comment by nullishamy** [comment]
Yeah, cool. What a ridiculous system. If the linked PR has no other issues we can close this one out, thanks for testing :+1:

**Comment by sgoudham** [comment]
FWIW, Pop OS has incorrect behaviour for both the `-dark` and `-Dark` suffixes that I've documented in https://github.com/catppuccin/gtk/pull/261#issuecomment-2136321111.

@nullishamy and I will probably have to create a `DISTRO.md` or something similar to point people towards to since it seems like each distro applies its own changes on top of GTK which can severely impact the user experience.

Dracula GTK users also experience this on Pop OS so we know it's a distro specific error -> https://github.com/dracula/gtk/issues/264

**Comment by wiraki** [comment]
Indeed, having the same problem on Pop_OS. 

It seems to me that with a previous version I did not have this problem, although I did have to use the `-dark` trick (known since 2021 from [here, an issue on the Nordic GTK theme](https://github.com/EliverLara/Nordic/issues/152#issuecomment-991772283)). But then all apps where themed properly, like evince, or tweaks itself. I don't remember which version it was, though. In any case, I am not sure that the GTK3 apps not theming has to do with the `-dark` suffix.

Speaking of https://github.com/EliverLara/Nordic in the past, how do they work around it? 

It is unclear, given #262, whether to wait for a fix. I wish I could contribute but I don't the skills.
